### PR TITLE
Feature/qti 128 support MS export

### PR DESCRIPTION
**Pull request by wtritch**
This PR adds support for converting TIMS MS items into QTI 3.0.

The bulk of the changes are in `MultipleSelectQtiMapper`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -378,8 +378,3 @@ task wrapper(type: Wrapper) {
 tasks.artifactoryPublish.dependsOn 'build'
 
 test.finalizedBy(jacocoTestReport)
-test {
-    jacoco {
-        excludes ['org/imsglobal/xsd/**']
-    }
-}
```
Comment (wtritch) [inline build.gradle]: Removed this, since it wasn't being respected.

File: src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/CommonQtiMapperSupport.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import static org.opentestsystem.ap.item.translation.mappers.qti.QtiConstants.OutcomeScore;
+import static org.opentestsystem.ap.item.translation.mappers.qti.QtiMappingUtils.getMaxPoints;
+
+/**
+ * This support service is responsible for providing standard mapping methods
+ * for all item types.
+ */
+@Component
+class CommonQtiMapperSupport {
```
Comment (wtritch) [inline src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/CommonQtiMapperSupport.java]: I broken out a common support bean for methods across many item types.  (Such as generating the same SCORE outcome declaration that all item types will need.)

File: src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/MultipleSelectQtiMapper.java [hunk 3]
```diff
@@ -23,7 +66,70 @@ public void map(final Item source, final QtiAssessmentItem target) {
...
+            .identifyOptions(msItem.getCore().getEn().getOptions());
+
+        target.getQtiResponseDeclarations().add(createResponseDeclaration(identifiedOptions));
+        target.getQtiOutcomeDeclarations().add(commonSupport.createScoreOutcomeDeclaration(msItem));
+
+        final Div contentRoot = contentMapper.appendContentRoot(target);
+        contentRoot.getContent().add(createPrompt(msItem));
+        contentRoot.getContent().add(createInteraction(msItem, identifiedOptions));
+        target.setQtiResponseProcessing(getResponseProcessing(msItem));
+    }
```
Comment (wtritch) [inline src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/MultipleSelectQtiMapper.java]: Here it is, in all it's glory.  Looks almost identical to the MultipleChoice mapper, with the fiddly-bits down in the implementations.

File: src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/MultipleSelectQtiMapper.java [hunk 4]
```diff
@@ -23,7 +66,70 @@ public void map(final Item source, final QtiAssessmentItem target) {
...
+        choiceInteraction.getQtiSimpleChoices()
+            .addAll(choiceSupport.mapOptions(item, identifiedOptions));
+        return choiceInteraction;
+    }
+
+    private long getMaxChoices(final MsItem item) {
+        final String maxChoicesMetadata = item.getCore().getMetadata().getMaxAllowedAnswers();
+        final long maxChoicesValue = isBlank(maxChoicesMetadata) ? 0 : Long.parseLong(maxChoicesMetadata, 10);
+        return maxChoicesValue > 0 ? maxChoicesValue : item.getCore().getEn().getOptions().size();
+    }
```
Comment (wtritch) [inline src/main/java/org/opentestsystem/ap/item/translation/mappers/qti/MultipleSelectQtiMapper.java]: This is respecting the "maxAllowedAnswers" metadata that allows a test-designer to prevent a student from selecting more than X options.

File: src/main/java/org/opentestsystem/ap/item/translation/rest/v1/ContentPackageApi.java [hunk 5]
```diff
@@ -68,7 +68,7 @@ public void deleteContentPackage(@PathVariable("id") final Long id) {
     public ResponseEntity<ContentPackageResource> getContent(@PathVariable("id") final Long id) {
         final ContentPackageResource resource = contentPackageService.getContent(id);
         return ResponseEntity.ok()
-            .header(CONTENT_DISPOSITION, "attachment; filename=\"content-pacakge-" + id + ".zip\"")
+            .header(CONTENT_DISPOSITION, "attachment; filename=\"content-package-" + id + ".zip\"")
```
Comment (wtritch) [inline src/main/java/org/opentestsystem/ap/item/translation/rest/v1/ContentPackageApi.java]: unrelated typo fix

File: src/test/java/org/opentestsystem/ap/item/translation/rest/v1/ContentPackageApiTest.java [hunk 6]
```diff
@@ -150,7 +150,7 @@ public void itShouldFetchContentPackageArchive() throws Exception {
 
             mvc.perform(get("/api/v1/content-packages/{id}/content", packageId))
                 .andExpect(status().isOk())
-                .andExpect(header().string(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=\"content-pacakge-" + packageId + ".zip\""))
+                .andExpect(header().string(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=\"content-package-" + packageId + ".zip\""))
```
Comment (wtritch) [inline src/test/java/org/opentestsystem/ap/item/translation/rest/v1/ContentPackageApiTest.java]: ...at least I was testing that the resource was mis-spelled =/

Comment (esotrope) [inline src/test/java/org/opentestsystem/ap/item/translation/rest/v1/ContentPackageApiTest.java]: lol
